setup/terminal.md

```diff
@@ -47,3 +47,58 @@ If you want to learn more about nvm
 ```
 nvm --help
 ```
+
+### Text editor from terminal level 
+
+It is good to also know some basic text editors available from terminal level.
+This is because your favorite graphical text editor might not always be installed 
+on someone else's computer while the terminal will always be there.
+Also if you work on a remote server machine the only thing which will be available to you will be the terminal.  
+The most popular one would be vi vim and nano. You do not have to be a super user and know all the shortcuts, 
```

Comment:
The most popular one would be vi, vim and nano. You do not have to be a super user and know all the shortcuts, 
but it can be extremely useful if you can to basic operations like